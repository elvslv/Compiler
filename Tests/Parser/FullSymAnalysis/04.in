var i, j: integer;
begin
	if i > j then
		i := j;
	else
		j := i;
end.