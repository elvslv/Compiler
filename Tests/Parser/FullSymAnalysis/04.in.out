var I : INTEGER;
var J : INTEGER;

5 5 Unexpected keyword found